tests/filecheck/dialects/func/func_invalid.mlir

```diff
@@ -36,4 +36,5 @@ builtin.module {
 
 "func.call"() { "callee" = @call::@invalid } : () -> ()
 
-// CHECK:  Operation does not verify: Unexpected nested symbols in FlatSymbolRefAttr.
+// CHECK:       Operation does not verify: Unexpected nested symbols in FlatSymbolRefAttr
+// CHECK-NEXT:  expected empty array, but got ["invalid"]
```

Comment:
```suggestion
// CHECK-NEXT:  Underlying verification failure: expected empty array, but got ["invalid"]
```
Maybe something like this to make it explicit?

Reply:
Done!